main() => throw StateError('Throwing for the heck of it');
